services/121-service/src/payments/services/transaction-jobs-creation.service.ts

```diff
@@ -178,6 +197,88 @@ export class TransactionJobsCreationService {
     );
   }
 
+  /**
+   * Creates and adds Intersolve Voucher transaction jobs.
+   *
+   * This method is responsible for creating transaction jobs for Intersolve Voucher. It fetches necessary PA data and maps it to a FSP specific DTO.
+   * It then adds these jobs to the transaction queue.
+   *
+   * @param {string[]} referenceIds - The reference IDs for the transaction jobs.
+   * @param {number} programId - The ID of the program.
+   * @param {number} paymentAmount - The amount to be transferred.
+   * @param {number} paymentId - The payment number.
+   * @param {boolean} isRetry - Whether this is a retry.
+   *
+   * @returns {Promise<void>} A promise that resolves when the transaction jobs have been created and added.
+   *
+   */
+  private async createAndAddIntersolveVoucherTransactionJobs({
+    referenceIdsAndTransactionAmounts: referenceIdsTransactionAmounts,
+    programId,
+    userId,
+    paymentId,
+    isRetry,
+    useWhatsapp,
+  }: {
+    referenceIdsAndTransactionAmounts: ReferenceIdAndTransactionAmountInterface[];
+    programId: number;
+    userId: number;
+    paymentId: number;
+    isRetry: boolean;
+    useWhatsapp: boolean;
+  }): Promise<void> {
+    const intersolveVoucherAttributes = getFspSettingByNameOrThrow(
+      Fsps.intersolveVoucherWhatsapp,
+    ).attributes;
+    const intersolveVoucherAttributeNames = intersolveVoucherAttributes.map(
+      (q) => q.name,
+    );
+    const dataFieldNames = [
+      FspAttributes.phoneNumber,
```

Comment:
Should phonenumber than not be an attribute at 
`services/121-service/src/fsps/fsp-settings.const.ts`

```
  {
    name: Fsps.intersolveVoucherPaper,
    integrationType: FspIntegrationType.api,
    defaultLabel: {
      en: 'Albert Heijn voucher paper',
    },
    notifyOnTransaction: false,
    attributes: [], // Add it here
```

Reply:
it is not needed. So removed this instead. And related, repeated the same exercise to remove 'phoneNumber' from SharedTransactionJobDto as you did in CBE PR. I saw btw that AirtelTransactionJobDto was not extending the SharedTransactionJobDto yet, while there seems no logical reason why not. Especially weird since I remember @nielsbom doing this refactor as part of the Airtel work. Anyway I updated this now, but let's discuss.